pkg/image/fetcher.go

```diff
@@ -246,3 +306,46 @@ func (w *colorizedWriter) Write(p []byte) (n int, err error) {
 	}
 	return w.writer.Write([]byte(msg))
 }
+
+func (i *ImagePullPolicyManager) UpdateImagePullRecord(path string, imageID string, timestamp string) error {
+	imageJSON, err := i.Read(path)
+	if err != nil {
+		return err
+	}
+
+	if imageJSON.Image.ImageIDtoTIME == nil {
+		imageJSON.Image.ImageIDtoTIME = make(map[string]string)
+	}
+	imageJSON.Image.ImageIDtoTIME[imageID] = timestamp
+
+	err = i.Write(imageJSON, path)
```

Comment:
@jjbustamante I think it's not a good idea to call `Read` and `Write` under every function. Should we handle it in the `Fetch` method only? What do you think?